src/utils/csvParser/parser.test.ts

```diff
@@ -0,0 +1,1544 @@
+/* eslint-disable @typescript-eslint/no-loss-of-precision */
```

Comment:
how many eslint errors are there related to loss of precision?

Reply:
Just one error. 
<img width="1132" alt="precision_loss_screenshot" src="https://user-images.githubusercontent.com/66275100/195626805-6273a8aa-38aa-4cc9-857b-49da69677cd6.png">
Instead of disabling eslint for the entire file, I can just disable it for this specific line.